testing/trino-testing/src/main/java/io/trino/testing/H2QueryRunner.java

```diff
@@ -105,7 +105,7 @@ public H2QueryRunner()
                 "  orderstatus CHAR(1) NOT NULL,\n" +
                 "  totalprice DOUBLE NOT NULL,\n" +
                 "  orderdate DATE NOT NULL,\n" +
-                "  orderpriority CHAR(15) NOT NULL,\n" +
+                "  orderpriority VARCHAR(15) NOT NULL,\n" +
```

Comment:
it's varchar in TPC-H, so it makes sense to change that as a prep commit.

btw same applies to orderstatus, clerk columns too